dev-libs/tvision/files/tvision-2.2.1.4-fix-abs.patch

```diff
@@ -0,0 +1,70 @@
+From 85185e7787b34545faf095b64935d80982b5cf81 Mon Sep 17 00:00:00 2001
```

Comment:
Could you combine both patches into a single file? It should be sufficient to concatenate them.

Reply:
I can do this, yes. I didn't it in the first place, because one patch is coming from upstream, while the other is coming from me. I notified upstream about the additional patch for the TDisplay class, but he didn't answer me so far.

Reply:
Well, either way is fine. I leave the decision to you.

Reply:
using a single file then